### DISCUSSION TITLE
Overflow design errors and too long sentences

### DISCUSSION THREAD

**Discussion by alekslyse** [firefly-iii/firefly-iii#8135] [General]
I have noticed one thing that is a recurring this and that is text is very limited to a full screen, at quite high resolution or overflow will happen very easily. This is even happening on the newer design. It might be cause I am on a mac, and have not tried on windows, but I suggest it would be better sizing control on the design, including making sure the description (for example the header in transactions that get unreadable on mobile. Since on IOS its not really any third part apps except the very limited abacus the design need some love on responsiveness.

Added some screenshots for reference

<img width="993" alt="Screenshot 2023-11-06 at 13 11 00" src="https://github.com/firefly-iii/firefly-iii/assets/3678663/0c9cf02d-b2ef-4d17-9069-54389e7ecff5">
<img width="995" alt="Screenshot 2023-11-06 at 13 08 00" src="https://github.com/firefly-iii/firefly-iii/assets/3678663/e7991da4-56c6-4f6a-8e42-982ff1c34518">
<img width="1218" alt="Screenshot 2023-11-06 at 13 07 37" src="https://github.com/firefly-iii/firefly-iii/assets/3678663/9ec1fc32-d505-4de0-a0d0-f709f1640299">

**Comment by JC5** [comment]
Yes I know. It's a limit of this layout. I hope to fix it in the new layout which has more options for this.

**Reply by alekslyse** [reply]
If you need any help on something please reach out. You have my email. Sr. Developer with 20+ years professional experience
